docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -213,6 +228,85 @@ To add a Task to the end of a Mission:
 
     :::
 
+### Custom Tasks  {#custom-task}
+
+Users can create their own tasks following a specific template and place them within the **/onav_log/custom_tasks** directory.
+Restarting the UGV or dockers will trigger the system to load the custom task that was created, making it available for mission use.
+Below is a sample template file named "log_inputs.py" which is elaborated further in the following section.
+
+#### Sample Custom Task
+
+```py
+#!/usr/bin/python3
+
+from onav_custom_tasks.custom_task_base import *
+
+class CustomTask(CustomTaskBase):

```

Comment:
Can we rename "CustomTask" to something else? It was unclear if this was code for an actual example or part of the base infrastructure. Maybe "ExampleCustomTask", "ExampleTask", "SampleCustomTask", etc.

Reply:
Currently these classes are always going to be "CustomTask(CustomTaskBase)" with the actual task name placed in the super().__init__ call.

Reply:
This has now been fixed and a new method is exposed for each module to allow for the user to name their classes accordingly.